projects/valtimo/plugin/src/lib/plugins/objecttypen-api/assets/objecttypen-api-plugin-logo.ts

```diff
@@ -0,0 +1,20 @@
W01QBwD2z6RoxfuCRffVvez3xo9e5Sd/qh/NyafFHnLG3+vuui7wuwYugTiff3eBNUqw+EMZorPMnMrhq8dEBlnpyOV+QHaZJA9g7SqeVIuWvRbtv8PQDLpClptpEgvz2ctM72Hy6PNcuC5rDSz3QUUyn0LDwEp+wUb8JtfjFgoUQClvIusOnDD5Ib4i3wOxk7Yd6Xo+70Ou5nN76Z1Lb3/nwqSA3A6HQCzWi5pQC7VgLFFLk4rzCHNyada2kT3vB/2tVBFs30kY9+QH1fNIhJ7cZI09+u0yTq8lgXu/fIFB15jPqa0RvZNKnx/dAbmc4/BiyUUMDq4M1Zsim9wHQaQC4X2AfQHNYGU7C2cm7UuY28QPrckF5YnxoG10zIpN3VWsCWBi704nxVahSXo2/Vk1taxJwP3mzwls9OyfQX7oDSp8HqUbSnf+wNRhQGBiyUYMBSXk4jd5sTpBajiQtzXR6yVp7fAMc+AddOR1nbImCnVV3t6/JGI5ev640yM0gu8BFSUzpY3s395RLQPOg9shz6tjznZpj2yWbcomYUGC/w1td+Vc6N9hWOQTRNrkKkTe8pQPu5BfY+MGChRANWBztt6J5wUhfCb850ZVi4VY6WO6tZLH83SdB5XWpHH0hAWQXHXSuPrVyu0gyxVdI66SNXc3+vs9yYXeWY3eWx7mak51X3Tcz2XsqdXF4X7t1Ild+GUXPfKvT5Yt5ylnUVmqbkvC0MEBS7NfI73Yh06lppBq5BYWHAQokHLOWNHLU1Hyg1J81truk5gs4hczcqymSKg65P89C7+BI5fArS5h4cNn9pMS62t2kAAAZESURBVE1s9fsLq4fKnxdJrfQYZK85Fgm7FMa5AM6AhwplBHkzBR2wYp9M111kPnyzpJn4PNr6SOByz5CvdXF0V+f9BAlMm9PULnO8QFXWOs2Mfi3J5G+R8YNr0wJ7f/XJGCi/i+tKLdboFlHF+LnVfsLHtCZuxsxfDAqENayYeH1MK9ZVS5Pue1I7+Lb8k/YtxZVeWWtjmuJ5Hoz7iFzm09Cv6vVC6UyPip/1oOwYiWT6mdBNNoulf0uTHV6Nlsr7Et5IIgg2CdENA9bG/CyTbV+VwHWEfKnbhelmClrzCnLBaZNOh991vs6b8ha+rItC5J18Cc32nTzkTkqcfz7TEyVMnw5jdf/CwlwOZbBM3i3NfnGzl2anODBgoZsHrI35Q/erd5IP8lAJYDVSIxoiLcBd5L/oRq3aYZ+phdl2qUmsk+ethhecdD6XqyN8S+CYJRLr3kLrmuV5SD9SMLwa1/qy0X6SQDMKXW9+x0U+7+YBOOk/4JkFC5OaexcRBiwwYG2V17E8c1QKn62uQJXUFNLr/FpXZbmL8pY02lrbsOzrbZg01S22TBBJ8Zrfq1rlBgDNrKF7AegmD0kvnJfmt9Ht1v+GlHsfZjW9UWSBqgM73Wnr/CDkddhnGTGKPc1QUcv01emu0/Ol1nUDWsr2hoNvSg1W53Fp8NKsCXHcnLX5rSsEZkqt90GY1JwCWFpT0hiwqKSYgxZrE1pzyD/rNb/XrN5Ngpeu/fuG3BpGyDN2lTqZZnHQpndXRmO1ttQixy6TwxZJ1fh5OPYZ2PQ8jF20grXfZDBgUcnythvzF3ZrudvbBLdH5bZwUzvBdSWQ2V2kFqaTUgfIs/t4E3Q3HK0ZE3SCpzZx3pPnLYWxbwLp95Da7pMCnUNV8hiwqNvIDFB8mCmbbJ3lZbUYNmlDs3HBVLv1PqhFoPxgwCJCJlEjpoIKW94zehIR5YoBi4iKBgMWERUNBiwiKhoMWERUNBiwiKhoMGARUdFgwCKiolHoAasniChJBT2ZPKmAFXQFOwMWUbL6hzg29vWVSQWsoFkwdwQRJSnMNbcCMUsqYH0W8LhBIKIkDQl4XCKpuZMKWO8HPE5zn/cBESVlWMDj3kUCkgpYCwMepz/fviCiJGj/1V4Bj30NCUgqYIVJIDQRRJSEo0Ic+yZKyGD421MFKc3gaCFREmYj+HX6A5SYDxD8ZJwFIoqTdr3YEKXkBsg0nWPQk7ESwbYJI6LOadfQiwh+fZZUc7CDbtceJoI/DaZ0JorDVQh3bV6BEqTBRvujwpyYO8GgRRSlMxHumtQyGCUqbCTXMg1sHhKFVSHlOoS/HmejhA2Uohtdhj1Jy6ScAmabIAriYCmvIvx1qOUglDht79qIymIpZ4Oz4Yk6o10p35TyJKK7/maiG9A5Va8jupOmpQ3+G3G+lP3BeVtE2vrQvqUTpdwlZTmiveZakIe+K4P82E/KHMTbpNOh1rfh7/L7EfzFmVOkzAVRablUSi8p28Hv3/0y/HW4VTF+z59JuR7dyIWINuLnUo4HUenRG3KS19ED6Kb9x7eBAYsorCQD1kvwa3PdknYE6twqBiyi4JIKWI3wR/q7Na1a3gAGLKKgkghYz4Gj8Zs4WcoaMGARdVXcAet2xNuBX7SGIroJbQxY1F3EFbA06cCxoKy0iajrm8KuO2TAou4i6oCVhl+r6g/KWT8plyPawMWARaUoqoClE7D/KmU4KDBtO58q5XkwYBFtSdiApZvE6MDXV0CR2hX+hFNdw9QKBiwiFSRg6TpcnQc5GkU0CTRfS3OioDWvA+Ev86mRUgt/bVNFlmO+D7/KS1RKNGBlS7mk6cnnS2mCvzStLvNvRaeYA9aW6J3iS5myM/w3sW/mv+0AriWk0qRrCbXWtBb+mllNv6TNvA8yZTWIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiAoT8H8LwFos5yAGTwAAAABJRU5ErkJggg==';
```

Comment:
I noticed this image is square with transparency at the top and bottom. Could you crop out this transparency (so there's no empty space in the logo image) and resize it to a maximum height of 60px as mentioned at https://valtimo.gitbook.io/valtimo-core-docs/extending-valtimo/plugins/custom-plugin-definition#plugin-logo